### DISCUSSION TITLE
集成管理和测试信息展示相关讨论

### DISCUSSION THREAD

**Discussion by hudeng-go** [linuxdeepin/developer-center#3787] [Q&A | 问答板块]
主要围绕[deepin-community/Repository-Integration](https://github.com/deepin-community/Repository-Integration)的pr开展集成流程，过程中涉及管理和测试方需要的集成信息或者和展示问题，可以在这个里面提供意见或需求

**Comment by hudeng-go** [comment]
- [x] 支撑bate集成的github pr流程完善
- [x] github pr集成信息同步到[v23集成看板](https://github.com/orgs/linuxdeepin/projects/21)中 (https://github.com/deepin-community/Repository-Integration/pull/71)
- [x] 集成到testing仓库的软件包同步到[aptly仓库](https://ci.deepin.com/repo/release/)，(方案更改：https://github.com/deepin-community/reprepro/pull/2 添加obs仓库同步支持)

**Reply by hudeng-go** [reply]
剩下两个任务有什么意见可以随时讨论。

**Reply by Zeno-sole** [reply]
> [ ] github pr集成信息同步到[v23集成看板](https://github.com/orgs/linuxdeepin/projects/21)中

第二条指的是什么？ Repository-Integration 下提交的集成 同步到集成看板中么？

**Reply by Zeno-sole** [reply]
第三条是要搞一下的，不然没法同步

**Reply by hudeng-go** [reply]
第二条就是指看板信息同步的。
